client/my-sites/site-settings/delete-site/index.jsx

```diff
@@ -45,6 +45,27 @@ module.exports = React.createClass( {
 		this.props.sites.off( 'change', this._updateSite );
 	},
 
+	renderNotice: function() {
+		var site = this.state.site;
+
+		if ( ! site ) {
+			return null;
+		}
+
+		return (
+			<Notice status="is-warning" showDismiss={ false }>
+				{ this.translate( '{{strong}}%(domain)s{{/strong}} will be unavailable in the future.', {
+					components: {
+						strong: <strong />
+					},
+					args: {
+						domain: site.domain
+					}
+				} ) }
+			</Notice>
+		);
```

Comment:
You could reduce the length of this function by doing the opposite - which I find more logic by the way:

```javascript
renderNotice: function() {
  var site = this.state.site;

  if ( site ) {
    return (
      <Notice status="is-warning" showDismiss={ false }>
        { this.translate( '{{strong}}%(domain)s{{/strong}} will be unavailable in the future.', {
          components: {
            strong: <strong />
          },
          args: {
            domain: site.domain
          }
        } ) }
      </Notice>
    );
  }
},
```